src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -24,6 +24,49 @@ public DataFrameColumn(string name, long length, Type type)
             DataType = type;
         }
 
+        #region Static Factory Methods
+
+        /// <summary>
+        /// A static factory method to create a <see cref="PrimitiveDataFrameColumn{T}"/>. 
+        /// It allows you to take advantage of type inference based on the type of the values supplied.
+        /// </summary>
+        /// <typeparam name="T">The type of the column to create.</typeparam>
+        /// <param name="name">The name of the column.</param>
+        /// <param name="values">The initial values to populate in the column.</param>
+        /// <returns>A <see cref="PrimitiveDataFrameColumn{T}"/> populated with the provided data.</returns>
+        public static PrimitiveDataFrameColumn<T> Create<T>(string name, IEnumerable<T?> values) where T : unmanaged
+        {
+            return new PrimitiveDataFrameColumn<T>(name, values);
+        }
+
+        /// <summary>
+        /// A static factory method to create a <see cref="PrimitiveDataFrameColumn{T}"/>. 
+        /// It allows you to take advantage of type inference based on the type of the values supplied.
+        /// </summary>
+        /// <typeparam name="T">The type of the column to create.</typeparam>
+        /// <param name="name">The name of the column.</param>
+        /// <param name="values">The initial values to populate in the column.</param>
+        /// <returns>A <see cref="PrimitiveDataFrameColumn{T}"/> populated with the provided data.</returns>
+        public static PrimitiveDataFrameColumn<T> Create<T>(string name, IEnumerable<T> values) where T : unmanaged
```

Comment:
The only difference to the above overload is `T?` vs `T`. 

So is
```suggestion
        public static PrimitiveDataFrameColumn<T> Create<T>(string name, IEnumerable<T> values) where T : unmanaged, notnull
```
useful?

Reply:
Microsoft.Data.Analysis is targeting .NET Standard 2.0 using C# 7.3, so the `notnull` constraint is not available.

Reply:
Oh, yes thanks for the hint!